frontend/app/src/types/notes.ts

```diff
@@ -50,7 +50,7 @@ export enum NoteLocation {
   PROFIT_LOSS_REPORTS = 'PROFIT_LOSS_REPORTS',
   ASSETS = 'ASSETS',
   PRICE_MANAGER = 'PRICE_MANAGER',
-  ETH_ADDRESS_BOOK_MANAGER = 'ETH_ADDRESS_BOOK_MANAGER',
+  ADDRESS_BOOK_MANAGER = 'ADDRESS_BOOK_MANAGER',
```

Comment:
What happens if you had notes in the old location?

Reply:
hmm yeah they won't see it. Should we keep the old name ?

Reply:
Or find a way to migrate from the old key to the new one.

Reply:
I think it's easier to migrate from backend.
Otherwise we can change only the enum name, but keep the value.